examples/nlp_seq/train.py

```diff
@@ -20,26 +20,25 @@
 import tensorflow as tf
```

Comment:
BTW : why did you remove `perplexity` ?

Reply:
It is unusual in part-of-speech tagging and many other tagging task. Further, I wanted the training loop more lean to make it more clear.